Use the component instance name when creating a matlab component interface

diff --git src/client.m src/client.m
index 8e5f758..1c11671 100644
--- src/client.m
+++ src/client.m
@@ -109,7 +109,7 @@ classdef client < handle
 
       module = genomix_server('get', self.id, ['/load/' client], args);
       meta = genomix_server('get', self.id,  ['/module/' module '/info']);
-      r = genomix.component(self, client, meta);
+      r = genomix.component(self, module, meta);
     end
 
 
